universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx

```diff
@@ -115,8 +116,16 @@ export const GasPrice = ({isDeployed = true, deployedWallet, sdk, gasLimit, onGa
       </div>
     </>
   );
+
+  const useClassForGasPrice = useClassFor('gas-price');
+
+  if (sdk.isRefundPaid()) {
+    onGasParametersChanged({gasToken: '0x0000000000000000000000000000000000000000', gasPrice: bigNumberify('0')});
```

Comment:
Extract `NO_REFUND_GAS_PARAMETERS` or `GAS_PRICE_FOR_NO_REFUND` or something like this